lib/web_ui/test/engine/scene_view_test.dart

```diff
@@ -101,4 +106,24 @@ void testMain() {
 
     debugOverrideDevicePixelRatio(null);
   });
+
+  test('SceneView always renders most recent picture', () async {
```

Comment:
nit: maybe add a test that shows that if you queue up 2 pictures (call them A and B) to be rendered while another picture is being rendered, that picture A is cancelled in favor of picture B

Reply:
That's sort of what this test is supposed to actually be covering, but it doesn't actually check to see if anything was canceled. I'm going to just go ahead and modify this test to more explicitly check for the cancelation part of it.